staging/src/k8s.io/apiserver/pkg/storage/value/metrics.go

```diff
@@ -41,6 +41,23 @@ var (
 		},
 		[]string{"transformation_type"},
 	)
+	transformerSuccessTotal = prometheus.NewCounterVec(
```

Comment:
I may be missing something in your comment.
Including the changes in this PR, we will have the following metrics:

transformation_success_total (this PR)
transformation_failures_total
transformation_latencies_microseconds
transformation_last_status (this PR)

Reply:
Sorry I failed to expand the github diff. success_total + failures_total + latencies are sufficient. I'd slightly prefer if you added a total usage counter rather than a success counter.

Reply:
Yes, in retrospect, I would have define a single counter - operations_total with two labels (success and failure).

Reply:
> Yes, in retrospect, I would have define a single counter - operations_total with two labels (success and failure).

It's not too late. If we're adding a metric here, let's add the one we want.

The current metric doesn't provide info about which resource type is being transformed (the feature is not limited just to `Secret` resources). Is that important to include?

Reply:
2 would be my preference, but I'm ok with deferring that until there is demand for it